pep/app/router.js

```diff
@@ -28,6 +28,12 @@ Router.map(function() {
 
     this.route('most-cited');
     this.route('most-viewed');
+    this.route('admin', function() {
+        this.route('common');
+        this.route('en-us');
+        this.route('fr-fr');
+        this.route('es-es');
```

Comment:
as mentioned above, wouldnt need individual routes defined w/the dynamic solution